build/images/scripts/install_cni_windows.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = "Stop";
+
+mkdir -force /host/var/run/secrets/kubernetes.io/serviceaccount
+cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/var/run/secrets/kubernetes.io/serviceaccount/
+mkdir -force /host/k/antrea/etc/
```

Comment:
Question - what is "k" in the path?

Reply:
`k` means `kubernetes`.

This dir is used by kubernetes windows official script to place k8s binaries.